### PR TITLE
s3-s3 implicit copy enhancements

### PR DESCRIPTION
**Pull request by shlomitk1**
This PR adds a M4DModule yaml for S3-S3 implicit copy and modifies a bucket information for testing.

### CODE REVIEW

**Review by froesef — APPROVED** [review]

File: manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  namespace: m4d-system  # In future, will be a parameter defined by admin
+  labels:
+    name: implicit-copy-s3-to-s3-batch # module name
+    version: 1.0.0 
+  annotations:
+    author: "Example Author (author@example.com)"
+    summary: "a short description of this module"
+    description: "a longer description of this module that might e.g. be used in contracts"
+spec:
+  type: service # one of: job, service, configuration
```
Comment (roee88) [inline manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml:18]: Job. 

We may want to omit this field from the spec, we don't seem to need it.

File: manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        dataformat: csv
+      sink: 
+        protocol: s3
+        dataformat: parquet
+    actions:
+    - id: redact-ID
+      level: 2  # column
+    - id: removed-ID
+      level: 2  # column
+  chart: ghcr.io/the-mesh-for-data/m4d-s3-to-s3:0.1.0
```
Comment (roee88) [inline manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml:36]: Where is this tag coming from?

File: manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+---
+apiVersion: app.m4d.ibm.com/v1alpha1
+kind: M4DModule
+metadata:
+  name: implicit-copy-s3-to-s3-batch # module name
+  namespace: m4d-system  # In future, will be a parameter defined by admin
+  labels:
+    name: implicit-copy-s3-to-s3-batch # module name
+    version: 1.0.0
```
Comment (roee88) [inline manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml:12]: We probably want to preserve consistency with the image tag

**Review by roee88 — COMMENTED** [review]
Minor comments

File: manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml (hunk 2 above)
Comment (shlomitk1) [reply manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml:36]: Chart.yaml of this module

File: manager/testdata/e2e/bucket-available.yaml [hunk 4]
```diff
@@ -4,9 +4,9 @@
 apiVersion: app.m4d.ibm.com/v1alpha1
 kind: M4DBucket
 metadata:
-  name: test-bucket
+  name: bucket-for-copy
 spec:
-  name: m4d-test-bucket
+  name: datamesh-test-bucket
```
Comment (ronenkat) [inline manager/testdata/e2e/bucket-available.yaml:9]: why are we using the term "datamesh"? and not "m4d"?

Comment (shlomitk1) [reply manager/testdata/e2e/bucket-available.yaml:9]: outdated artifacts...

**Comment by shlomitk1** [comment]
The comments reveal issues in some other modules as well. I'm closing this PR, will open a more general one.

**Pull request closed without merging**